doc/inference_parameters.md

```diff
@@ -0,0 +1,25 @@
+# ffn inference parameters
+
+The inference part of ffn reads parameters in a protobuf text format (pbtxt) and is
+normally invoked in a way that reads in such a file from the commandline. These are the
+parameters and what they mean:
+
+* image - A hash, usually with a single k-v pair of the string 'hdf5' to a path to an hdf5 file and dataset. Holds the raw image data against which inference is to be done
+* image_mean - In theory, the mean of values in the targeted region. In practice, leave this at 128
+* image_stddev - In theory, the standard deviation of values in the targeted region. In practice, leave this at 33
+* checkpoint_interval - How often inference should dump its internal state to disk. If inference is interrupted, these checkpoints allow for interrupted work to be picked back up with the same parameters. In some environments this is more important so a lower value is warranted
+* seed_policy - Leave as "PolicyPeaks"
+* model_checkpoint_path - The path to the files that comprise the model used. Omit the final extension; if there are many versions of the model in the same directory you will probably want to pick the highest (most trained) number.
```

Comment:
It's actually not completely correct that the highest numbered checkpoint is always the best. I believe we briefly mentioned this in the paper, but it probably deserves a more detailed discussion in a separate doc.